app/models/move.rb

```diff
@@ -83,6 +83,7 @@ class Move < VersionedModel
   has_many :journeys, -> { default_order }, dependent: :restrict_with_exception, inverse_of: :move
   has_many :court_hearings, dependent: :restrict_with_exception
   has_many :move_events, as: :eventable, dependent: :destroy # NB: polymorphic association
+  has_many :documents
```

Comment:
This relationship was missing - documents can belong to moves optionally, and the front end download code looks to be counting move documents so I matched that behaviour.

Reply:
Documents are now on a profile

Reply:
Will revert this 👍